website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -16,6 +16,58 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### Deprecated Agent Config Options
+
+The `master` and `agent_master` ACL tokens in the `acl.tokens` config block
```

Comment:
Is it worth mentioning that the `/agent/token/agent_master` endpoint [has been deprecated](https://github.com/hashicorp/consul/blob/f76fc6cabeab1d2c6673f00bc95f215cf787f371/website/content/api-docs/agent/index.mdx?plain=true#L742-L747) too?